numpy/core/src/multiarray/textreading/tokenize.cpp

```diff
@@ -1,20 +1,16 @@
 
 #include <Python.h>
 
-#include <stdio.h>
```

Comment:
if you need them, these files still exist in C++ as `<cstdio>, <cstdlib> and <cstring>` `stdbool.h` is not super useful in C++

Reply:
Yeah, but seems none of them were needed anymore, I guess I had used `memcpy`, etc. earlier but then replaced them.

Reply:
ok, great!